composer.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "utopia-php/compression",
+    "description": "A simple Compression library to handle file compression",
+    "type": "library",
+    "keywords": ["php","framework","upf","utopia","compression"],
+    "license": "MIT",
+    "authors": [],
+    "autoload": {
+        "psr-4": {"Utopia\\Compression\\":"src/Compression"}
+    },
+    "autoload-dev": {
+        "psr-4": {"Utopia\\Tests\\Compression\\":"tests/Compression"}
+    },
+    "scripts": {
+        "lint": "./vendor/bin/pint --test",
+        "format": "./vendor/bin/pint"
+    },
+    "require": {
+        "ext-fileinfo": "*",
+        "ext-zlib": "*",
+        "ext-zstd": "*",
+        "ext-xz": "*",
+        "ext-brotli": "*",
+        "ext-lz4": "*",
+        "ext-snappy": "*",
+        "php": ">=8.0"
+    },
```

Comment:
this will end up requiring all of those extensions to be present, even if you dont use them.

in `utopia-php/database` we are using the `suggest` key 👍🏻 

https://github.com/utopia-php/database/blob/main/composer.json#L53-L58

Reply:
More context here https://getcomposer.org/doc/04-schema.md#suggest